### PR TITLE
Fix LabeledItem to pass marqueeOn prop

### PR DESCRIPTION
**Pull request by webOS101**
Enact-DCO-1.0-Signed-off-by: Roy Sutton roy.sutton@lge.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
`LabeledItem` had no way to adjust when it would marquee. A use case existed where
it was used as a non-focusable component, so the default `marqueeOn` of `"focus"`
prevented it from being able to marquee.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Pass `marqueeOn` prop to label and contents.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Another possible solution is to modify the component to detect `spotlightDisabled` and automatically set `marqueeOn` for the items.  This seemed a bit too magical and we'd need to make a similar change to all focusable components for consistency. This solution makes a very small change and gives developers the tools they need to accomplish the use case desired.

I decided to not pass along any additional marquee properties as they were not relevant to this request.

### Links
[//]: # (Related issues, references)
ENYO-6100

### CODE REVIEW

File: packages/ui/Marquee/MarqueeDecorator.js [hunk 1]
```diff
@@ -226,9 +226,7 @@ const MarqueeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			/**
 			 * Determines what triggers the marquee to start its animation.
 			 *
-			 * Valid values are `'focus'`, `'hover'` and `'render'`. The default is `'focus'`.
-			 *
-			 * @type {String}
+			 * @type {('focus'|'hover'|'render')}
```
Comment (Djspaceg) [inline packages/ui/Marquee/MarqueeDecorator.js]: Do we render out the possible values of typescript in our docs for cases like this? If so, neat!

Comment (webOS101) [reply packages/ui/Marquee/MarqueeDecorator.js]: Something like that.  We do render the values into our typescript def. files and we do render out the array of possible values into the docs.  I think it's much more descriptive than just 'String'

Comment (Djspaceg) [reply packages/ui/Marquee/MarqueeDecorator.js]: It is indeed much more descriptive. I'm just verifying that we're not _losing_ documentation/description on our website. Whether, for example, someone who doesn't use typescript would be able to read the list of acceptable values in some way.

Comment (webOS101) [reply packages/ui/Marquee/MarqueeDecorator.js]: Nope, we don't lose anything by this.  The docs will list the acceptable values.

Comment (ryanjduffy) [reply packages/ui/Marquee/MarqueeDecorator.js]: docs
image

.d.ts
image

codesandbox.io
image

Comment (Djspaceg) [reply packages/ui/Marquee/MarqueeDecorator.js]: Cool. Thanks for sharing the pictures. They look different from what I was seeing.

**Comment by ryanjduffy** [comment]
Did you investigate expanding support for `marqueeOn` in `MarqueeController`? I wonder if it should have API compatibility with `MarqueeDecorator` as it behaves as a proxy for multiple marquees.

I'm good with the current approach as is as expanding `MarqueeController` would be compatible with this API but curious if we should investigate further separately.

**Comment by webOS101** [comment]
I did think about it briefly but there is already some weird coupling between the controller and the decorator and it seemed like teasing that out right now would possibly introduce a big chance of regression.  Marquee is already one of the biggest sources of bugs and simplifying the implementation would always be great... but right now didn't seem like the time.

**Comment by webOS101** [comment]
By coupling, I meant overlapping concerns.  Of course they're coupled since Controller is supposed to control Decorator.

**Pull request merged**